source/Radar.cpp

```diff
@@ -51,7 +52,9 @@ void Radar::SetCenter(const Point &center)
 }
 
 
-
+// Planets and stars both are circles, so they have "inner" as 0, but I can't
+// tell where it sets that. This may be clue to find where the identity of an object
+// is identified as a star.
 // Add an object. If "inner" is 0 it is a dot; otherwise, it is a ring. The
 // given position should be in world units (not shrunk to radar units).
 void Radar::Add(int type, Point position, double outer, double inner)
```

Comment:
inner is set here, whenever the function is called for a planet,
engine.cpp line 2271

Reply:
depending on which editor you use, you can right click on a function name and select find all references to find out where it is used

Reply:
NP++ And yes, it does have a "find all in folder" option that I have been using extensively to look for key terms and things.